dts/arm/ambiq/ambiq_apollo4p.dtsi

```diff
@@ -206,6 +207,61 @@
 		pinctrl: pin-controller@40010000 {
 			compatible = "ambiq,apollo4-pinctrl";
 			reg = <0x40010000 0x800>;
+			#address-cells = <1>;
+			#size-cells = <0>;
+
+			gpio: gpio@40010000 {
+				compatible = "simple-bus";
```

Comment:
?

Reply:
Thanks for your catching this. Did cleanup.